### PR TITLE
Routes and templates

### PR DESCRIPTION
**Pull request by kellywerks**
Part of #445 Add tutorial for building first Ember app

Note:  images when viewing the .md file in Github appear to be broken but are not.  In Middleman and raw code, links are correct.

### CODE REVIEW

File: source/getting-started/routes-and-templates.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+</p>
+
+<p>(503)555-1212</p>
+
+<p>superrentalsrep@superrentals.com</p>
+```
+
+Now we have completed our second static page.  If we enter the URL,  `http://super-rentals/contact`, we arrive on our contact page.
+
+## Navigating with links and the {{link-to}} helper
```
Comment (michaelrkn) [inline source/getting-started/routes-and-templates.md]: Titles should be title-cased

File: source/getting-started/routes-and-templates.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+</p>
+
+<p>(503)555-1212</p>
+
+<p>superrentalsrep@superrentals.com</p>
+
+{{#link-to 'about'}}About{{/link-to}}
+```
+
+## An Index Page
```
Comment (michaelrkn) [inline source/getting-started/routes-and-templates.md]: How about "An Index Route"?

Comment (kellywerks) [inline source/getting-started/routes-and-templates.md]: Done.  Also updated other "Page" titles to "Route" (About and Contact).

File: source/getting-started/routes-and-templates.md [hunk 3]
```diff
@@ -0,0 +1,166 @@
+To demonstrate the basic setup and processing of an Ember application, the next few lessons are going to walkthrough building a very simple application for a property rental site called Super Rentals.  It will start with just a homepage, an about page and a contact page.  Let's take a look at the application from the user perspective before we get started.
```
Comment (locks) [inline source/getting-started/routes-and-templates.md]: `are going to walkthrough building` is a strange turn of phrase

Comment (locks) [inline source/getting-started/routes-and-templates.md]: `a very simple application` is a bit strong

File: source/getting-started/routes-and-templates.md [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+
+We arrive at the home page which shows a list of rentals.  From here, we will be able to navigate to an about page and a contact page.  
+
+We already have the new app called super-rentals created. Before we start building the three pages for our app, we are going to clear out the contents of the `app/templates/application.hbs` file and only leave the {{outlet}} code in place.  We'll talk more about the role of the `application.hbs` file after our site has a few pages.
+
+Now, let's start with building our "about" page. Remember, when the URL path `/about` is loaded, the router will map the URL to the route handler of the same name, _about.js_.  The route handler then loads a template.  
+
+## An About Route
+
+If we run `ember help generate`, we can see a variety of tools come with Ember for automatically generating files for various Ember resources.  We'll use the route generator to build our `about` route.
```
Comment (locks) [inline source/getting-started/routes-and-templates.md]: Is this the first time `ember generate` is referenced?

File: source/getting-started/routes-and-templates.md [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+
+```shell
+ember generate route about
+```
+
+or for short,
```
Comment (locks) [inline source/getting-started/routes-and-templates.md]: I'd remove this part

File: source/getting-started/routes-and-templates.md [hunk 6]
```diff
@@ -0,0 +1,166 @@
...
+<h2>About Super Rentals</h2>
+
+<p>The Super Rentals website is a delightful project created to explore Ember.<br>
+  By building a property rental site, we can simultaneously imagine traveling<br>
+  AND building Ember applications simultaneously.</p>
+```
+
+## A Contact Route
+
+Let's create another static page with details for contacting the company.  Once again, we'll start by generating a route, a route handler, and a template.
```
Comment (locks) [inline source/getting-started/routes-and-templates.md]: I'd refrain from calling them static pages, as that would imply they're page.html files in `/public`.

File: source/getting-started/routes-and-templates.md [hunk 7]
```diff
@@ -0,0 +1,166 @@
...
+
+```shell
+ember g route about
```
Comment (locks) [inline source/getting-started/routes-and-templates.md]: should be `route contact`

File: source/getting-started/routes-and-templates.md [hunk 8]
```diff
@@ -0,0 +1,166 @@
...
+
+{{#link-to 'about'}}About{{/link-to}}
+```
+
+## An Index Route
+
+With our two static pages in place, we are ready to add our home page which welcomes users to the site.  Using the same process we did for our about and contact pages, we will first generate a new route called "index".
+
+```shell
+ember g route index
```
Comment (locks) [inline source/getting-started/routes-and-templates.md]: missing a `>` or `$` to indicate a user command

File: source/getting-started/routes-and-templates.md [hunk 9]
```diff
@@ -0,0 +1,166 @@
...
+version: 1.13.8
+installing route
+  create app/routes/index.js
+  create app/templates/index.hbs
+installing route-test
+  create tests/unit/routes/index-test.js
+```
+
+Unlike the other route handlers we've made so far, the `index` route is special: it does NOT require an entry in the router's mapping, and you'll notice that the action of updating the router was not taken by the generator. We'll learn more about why the entry isn't required when we touch on nested routes.
```
Comment (locks) [inline source/getting-started/routes-and-templates.md]: linking to the guides page where it talks about `index` might be an alternative

File: source/getting-started/routes-and-templates.md [hunk 10]
```diff
@@ -0,0 +1,166 @@
+To demonstrate the basic setup and processing of an Ember application, the next few lessons are going to walkthrough building a very simple application for a property rental site called Super Rentals.  It will start with just a homepage, an about page and a contact page.  Let's take a look at the application from the user perspective before we get started.
+
+![super rentals homepage screenshot](../../images/routes-and-templates/ember-super-rentals-index.png)
+
+We arrive at the home page which shows a list of rentals.  From here, we will be able to navigate to an about page and a contact page.  
+
+We already have the new app called super-rentals created. Before we start building the three pages for our app, we are going to clear out the contents of the `app/templates/application.hbs` file and only leave the {{outlet}} code in place.  We'll talk more about the role of the `application.hbs` file after our site has a few pages.
```
Comment (steveklabnik) [inline source/getting-started/routes-and-templates.md]: Where was this created? If I'm starting here, and I didn't create this app, where should I go to create it?

File: source/getting-started/routes-and-templates.md (hunk 6 above)
Comment (steveklabnik) [inline source/getting-started/routes-and-templates.md]: :+1:

File: source/getting-started/routes-and-templates.md [hunk 11]
```diff
@@ -0,0 +1,166 @@
...
+  1212 Test Address Avenue<br>
+  Testington, OR 97233
+</p>
+
+<p>(503)555-1212</p>
+
+<p>superrentalsrep@superrentals.com</p>
+```
+
+Now we have completed our second static page.  If we enter the URL,  `http://super-rentals/contact`, we arrive on our contact page.
```
Comment (steveklabnik) [inline source/getting-started/routes-and-templates.md]: Does this URL work? How did it get set up?

File: source/getting-started/routes-and-templates.md [hunk 12]
```diff
@@ -0,0 +1,166 @@
...
+
+## Navigating with Links and the {{link-to}} Helper
+
+We really don't want users to have to know our URLs in order to move around our site, so let's add some navigational links at the bottom of each page.  Let's make a contact link on the about page and an about link on the contact page.
+
+Ember has built-in **helpers** that provide functionality such as linking to other routes.  Here is how to use the {{link-to}} helper in our code:
```
Comment (steveklabnik) [inline source/getting-started/routes-and-templates.md]: graves are needed so `{{link-to}}` gets rendered like code

File: source/getting-started/routes-and-templates.md (hunk 10 above)
Comment (locks) [inline source/getting-started/routes-and-templates.md]: I believe https://github.com/emberjs/guides/pull/745 comes before, and is where the app is generated.

Comment (locks) [inline source/getting-started/routes-and-templates.md]: Unlike the guides, where we try to sort of make them standalone, and link freely, the tutorial has the expectation that the user will go from one end to the other. Any suggestion on addressing your question?

Comment (steveklabnik) [inline source/getting-started/routes-and-templates.md]: Ah, right. So yeah, maybe just link to that here?

File: source/getting-started/routes-and-templates.md (hunk 8 above)
Comment (michaelrkn) [inline source/getting-started/routes-and-templates.md]: I don't believe anywhere else in the guides prefixes a shell prompt.

**Pull request closed without merging**